metadata-service/factories/src/main/resources/application.yml

```diff
@@ -150,6 +150,7 @@ elasticsearch:
     numShards: ${ELASTICSEARCH_NUM_SHARDS_PER_INDEX:1}
     numReplicas: ${ELASTICSEARCH_NUM_REPLICAS_PER_INDEX:1}
     numRetries: ${ELASTICSEARCH_INDEX_BUILDER_NUM_RETRIES:3}
+    refreshIntervalSeconds: ${ELASTICSEARCH_INDEX_BUILDER_REFRESH_INTERVAL_SECONDS:30}
```

Comment:
I think we should keep 1 the default here, most deployments are not large enough to hit the write scaling issues that need the 30s setting.